pandas/tests/tseries/offsets/test_offsets.py

```diff
@@ -121,17 +156,19 @@ class TestCommon(Base):
         "Nano": Timestamp("2011-01-01T09:00:00.000000001"),
     }
 
+
+class TestCommon:
     def test_immutable(self, offset_types):
         # GH#21341 check that __setattr__ raises
-        offset = self._get_offset(offset_types)
+        offset = _create_offset(offset_types)
```

Comment:
shouldn't you make a *new* fixture that does this

Reply:
This function is also used in the testing class helper methods so I think this is better (for now) as a separate function